content/docs/start/data-management/data-versioning.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Get Started: Data Versioning'
 description: 'Get started with data and model versioning in DVC. Learn how to
-use a regular Git workflow for datasets and ML models, without storing large
+use a standard Git workflow for datasets and ML models, without storing large
```

Comment:
since there's no _one_ Git workflow standard, we can just say "a Git workflow" + link to something like https://www.atlassian.com/git/tutorials/comparing-workflows
or https://about.gitlab.com/topics/version-control/what-is-git-workflow/

Reply:
We're on the same page. This reads "a std Git workflow" implying that there are multiple standard ones. Does it sound like "the std git flow" ? I feel like without "standard", people who aren't familiar with Git workflows won't have any idea of why that's relevant. But not a strong opinion: if just "a Git workflow" is easier to read we should probably go with that.

Reply:
I guess we can leave it as is because I am not sure whether we should optimize for someone familiar with git workflows or someone who's not.